x/group/server/invariants.go

```diff
@@ -12,13 +12,15 @@ import (
 )
 
 const (
-	votesInvariant  = "Tally-Votes"
-	weightInvariant = "Group-TotalWeight"
+	votesInvariant              = "Tally-Votes"
+	weightInvariant             = "Group-TotalWeight"
+	proposalTallyVotesInvariant = "Tally-Proposal-Votes-With-Sum-Of-Votes"
```

Comment:
what about this instead?
```suggestion
	VotesSumInvariant = "Tally-Votes-Sum"
```